cymem/cymem.pxd

```diff
@@ -1,4 +1,6 @@
 ctypedef void* (*malloc_t)(size_t n)
+ctypedef void* (*calloc_t)(size_t n)
+ctypedef void* (*realloc_t)(size_t n)
```

Comment:
If you're adding to the public API, you should explain in the PR description why. Also, `calloc` method is added to the implementation by not here.